### PR TITLE
Add support for os.detected.version.{major, minor} properties

### PR DESCRIPTION
**Pull request by marko-asplund**
address part of issue #15

### CODE REVIEW

File: src/main/java/kr/motd/maven/os/Detector.java [hunk 1]
```diff
@@ -64,6 +72,12 @@ protected void detect(Properties props, List<String> classifierWithLikes) {
         setProperty(props, DETECTED_NAME, detectedName);
         setProperty(props, DETECTED_ARCH, detectedArch);
 
+        Matcher versionMatcher = VERSION_REGEX.matcher(osVersion);
+        if (versionMatcher.matches()) {
+            setProperty(props, DETECTED_VERSION_MAJOR, versionMatcher.group(1));
+            setProperty(props, DETECTED_VERSION_MINOR, versionMatcher.group(2));
```
Comment (trustin) [inline src/main/java/kr/motd/maven/os/Detector.java]: Could we also make `DETECTED_VERSION` public and set the full version string? (please update the documentation as well)

**Comment by trustin** [comment]
@marko-asplund Sorry for taking so long to respond to this PR. I wasn't watching this project actively. I left a comment. Could you address it?

**Comment by marko-asplund** [comment]
@trustin Thanks for your review! I just updated the pull request to address your comments.

File: src/main/java/kr/motd/maven/os/DetectExtension.java [hunk 2]
```diff
@@ -45,6 +45,9 @@
  * <ul>
  * <li>{@code os.detected.name} - normalized {@code os.name} (e.g. {@code linux}, {@code osx})</li>
  * <li>{@code os.detected.arch} - normalized {@code os.arch} (e.g. {@code x86_64}, {@code x86_32})</li>
+ * <li>{@code os.detected.version.major} - major part of {@code os.version} (integer value)</li>
+ * <li>{@code os.detected.version.minor} - minor part of {@code os.version} (integer value)</li>
+ * <li>{@code os.detected.version} - {@code os.detected.version.major}.{@code os.detected.version.minor}</li>
```
Comment (trustin) [inline src/main/java/kr/motd/maven/os/DetectExtension.java]: Could you move this before major and minor?

**Comment by trustin** [comment]
One last comment. We should be good to go once it's fixed. Thanks again for your patience!

File: pom.xml [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
   <groupId>kr.motd.maven</groupId>
   <artifactId>os-maven-plugin</artifactId>
-  <version>1.4.2.Final-SNAPSHOT</version>
+  <version>1.4.3-SNAPSHOT</version>
```
Comment (trustin) [inline pom.xml]: Could you revert this?

**Comment by marko-asplund** [comment]
reverted.

File: src/main/java/kr/motd/maven/os/DetectExtension.java (hunk 2 above)
Comment (trustin) [inline src/main/java/kr/motd/maven/os/DetectExtension.java]: Ugh, never mind.

**Comment by trustin** [comment]
I think we are all set. Could you please squash the commits into one?

**Comment by marko-asplund** [comment]
yes, done.

**Comment by trustin** [comment]
Merged. Will release when time allows. Thanks again!

**Comment by marko-asplund** [comment]
excellent, thanks a lot!

**Pull request merged**